deployments/run.sh

```diff
@@ -48,17 +52,28 @@ do
     --no-monitoring)
       MONITORING=" -f ./monitoring/disabled.yml"
       shift
+      continue
       ;;
     poa-rialto)
-      BRIDGE=$i
+      BRIDGES+=($i)
       NETWORKS+=${RIALTO}
+      RIALTO=''
```

Comment:
What are these network-specific variables about?

Reply:
They're defined before the loop. This 'nullifying' is needed to avoid duplicating the same (Millau|Rialto) nodes